src/foam/mlang/mlang.js

```diff
@@ -2751,7 +2751,7 @@ foam.CLASS({
     {
       name: 'put',
       code: function put(o, sub) {
-        var a = [o.cls_];
+        var a = [o.cls_.id];
```

Comment:
or can do check for a[0] being instance of class on line 2717

Reply:
I don't remember, but probably on line 2717. I changed line 2754 to keep it consistent with backend

Reply:
but I can change 2754 instead